Dockerfile.dev

```diff
@@ -6,6 +6,6 @@ RUN npm i -g pnpm
 
 COPY . /usr/src/wowanalyzer/
 
-EXPOSE 3000
```

Comment:
would prob be good to keep the development port the same if possible to easily/reliably configure. we have enough stuff changing, don't need to add more.

Reply:
I will repoint everything to 3000